hack/community/readme.md

```diff
@@ -1,21 +1,49 @@
 # Cut a new community release
-The generate.sh script is called in the pre step to set up for the 
-aws-e2e-operator-test. The script copies over the WMCO community 
-bundle to the artifact directory while also replacing csv fields such as the 
-name, description, and date. Once generated into the artifact directory, the 
-bundle is used in the windows-e2e-operator test. After it passes, the bundle can
-be used to release a new community WMCO.
+CI contains a COMMUNITY environment variable that defaults to false, and is only
+set to true when a community-4.x branch PR is opened. When true, CI is 
```

Comment:
`contains a COMMUNITY environment variable` should be reworded, as thats not using fully accurate terminology.

consider something like this: 
```
In CI jobs the `COMMUNITY` environment variable is set to false by default, and is set true when testing PRs against community-4.x branches.
```